src/features/keyboard.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Creating a new keyboard.json is something we do for extensions. Since FindReferences is a core feature, you should add this to the main [keyboard.json](https://github.com/adobe/brackets/blob/master/src/base-config/keyboard.json).

Change the key to "cmd.findAllReferences".

Additionally, add the menu entries under [DefaultMenus.js](https://github.com/adobe/brackets/blob/master/src/command/DefaultMenus.js) instead of in FindReferencesManager.js

Also, you'll need to add it at two places, one is the context menu [here](https://github.com/adobe/brackets/blob/master/src/command/DefaultMenus.js#L281) and secondly in the AppBarMenu [here](https://github.com/adobe/brackets/blob/master/src/command/DefaultMenus.js#L138), below FIndInFiles